### ISSUE TITLE
Command line tool always exits as successful (exit code 0)

### ISSUE THREAD

**Issue by rbanffy** [paazmaya/yaml-validator#21]
### Expected behaviour

When invalid files are encountered, an exit code other than 0 should be returned.

### Actual behaviour

Running the command line tool on an invalid file results in a success (exit code 0).

```sh
$ yaml-validator valid.yaml
$ echo $?
0
$ yaml-validator invalid.yaml
end of the stream or a document separator is expected at line 8, column 1:
    - {"name":"crm_id","type":"STRIN ...
    ^
$ echo $?
0
```

What should happen is:

```sh
$ yaml-validator invalid.yaml
end of the stream or a document separator is expected at line 8, column 1:
    - {"name":"crm_id","type":"STRIN ...
    ^
$ echo $?
1
```

With a non-zero result (1 in this example because it's a single file).

### Versions and environment

 * Operating system: macOS
 * Node.js: v8.15.0
 * npm: 6.5.0
 * yaml-validator: fresh checkout from master

Thank you and have some :grapes:.

**Comment by paazmaya** [comment]
Good suggestion, perhaps the exit code could be the number of failed files

**Comment by paazmaya** [comment]
Released in `v2.0.0` https://www.npmjs.com/package/yaml-validator
